### PR TITLE
Add ur translation

### LINKED ISSUE #97: Add support for Urdu language
@wojtekmaj 
Urdu is currently missing from the available languages dropdown. I have added Urdu language support.

### PR DESCRIPTION
**Pull request by rabbanirizwan**
Closes #97

### CODE REVIEW

File: src/i18n/i18n.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const languageFiles = {
   'th-TH': () => import(/* webpackChunkName: "th-TH" */ './json/th-TH.json'),
   'tr-TR': () => import(/* webpackChunkName: "tr-TR" */ './json/tr-TR.json'),
   'uk-UA': () => import(/* webpackChunkName: "uk-UA" */ './json/uk-UA.json'),
+  'ur': () => import(/* webpackChunkName: "uk-UA" */ './json/ur.json'),
```
Comment (wojtekmaj) [inline src/i18n/i18n.js:26]:
```suggestion
  'ur': () => import(/* webpackChunkName: "ur" */ './json/ur.json'),
```

File: src/i18n/json/ur.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "Mounting": "ماؤنٹنگ",
+  "Updating": "اپ ڈیٹ کر رہا ہے۔ DOM",
+  "Unmounting": "اَن ماؤنٹنگ",
+  "“{name} phase”": "{name} کا مرحلہ",
+  "Pure and has no side effects. May be paused, aborted or restarted by React.": " رد عمل کے ذریعے روکا، ساقط یا دوبارہ شروع کیا جا سکتا ہے۔     خالص اور کوئی سائیڈ ایفیکٹ نہیں ہے",
+  "Can read the DOM.": "پڑھ سکتے ہیں۔ DOM.",
+  "Can work with DOM, run side effects, schedule updates.": " کے ساتھ کام کر سکتے ہیں، سائیڈ ایفیکٹس، شیڈول اپ ڈیٹس چلا سکتے ہیں۔  DOM",
+  "React updates DOM and refs": "کو اپ ڈیٹ کرتا ہے۔refs اور DOM React",
+  "Read docs for {name} (opens in a new tab)": "کے لیے دستاویزات پڑھیں (ایک نئے ٹیب میں کھلتا ہے) {name}",
+  "//reactjs.org/docs/react-component.html#{docname}": "//vi.reactjs.org/docs/react-component.html#{docname}",
```
Comment (wojtekmaj) [inline src/i18n/json/ur.json:16]: Sure you want links to Vietnamese docs? :)

```suggestion
  "//reactjs.org/docs/react-component.html#{docname}": "//ur.reactjs.org/docs/react-component.html#{docname}",
```

File: src/i18n/languages.json [hunk 3]
```diff
@@ -119,6 +119,11 @@
     "name": "Ukrainian",
     "translated_name": "Українська"
   },
+  {
+    "code": "ur",
+    "name": "اردو",
```
Comment (wojtekmaj) [inline src/i18n/languages.json:124]:
```suggestion
    "name": "Urdu",
```

**Review by wojtekmaj — CHANGES_REQUESTED** [review]
Almost there :)

File: src/i18n/json/ur.json (hunk 2 above)
Comment (rabbanirizwan) [reply src/i18n/json/ur.json]: I have changed it to eng because at the moment react does not support Urdu

File: src/i18n/i18n.js (hunk 1 above)
Comment (rabbanirizwan) [reply src/i18n/i18n.js]: resolve

File: src/i18n/languages.json (hunk 3 above)
Comment (rabbanirizwan) [reply src/i18n/languages.json]: resolve

File: src/i18n/json/ur.json (hunk 2 above)
Comment (wojtekmaj) [reply src/i18n/json/ur.json]: Hmmm, locale is there, live: https://ur.reactjs.org/

**Review by wojtekmaj — APPROVED** [review]

**Pull request merged**